src/components/dropdown-menu/DropdownMenuItemButtonDefault.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { DropdownMenuItemBase, DropdownMenuItemBaseProps } from './DropdownMenuItemBase';
+
+type Props = Omit<DropdownMenuItemBaseProps, 'isCheckable' | 'isChecked' | 'isDestructive'>;
+
+export function DropdownMenuItemButtonDefault(props: Readonly<Props>) {
```

Comment:
Let's name it `DropdownMenuItemButton` ?

Reply:
yes I was thinking the same thing yesterday